docs/src/main/asciidoc/security-getting-started-tutorial.adoc

```diff
@@ -4,39 +4,35 @@ and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
 [id="security-getting-started-tutorial"]
-= Getting Started with Security using Basic authentication and Jakarta Persistence
+= Getting started with security using Basic authentication and Jakarta Persistence
```

Comment:
Can we keep `Security` ? It is a well known term in Quarkus, same as `Basic`, I believe we have a few `Security` words, `security` does not read as a dedicated area in Quarkus.  cc @rsvoboda

Reply:
Or may be it is OK, not sure, I appreciate the RH standards requiring the lower case for regular words, but it seems like it is not

Reply:
Looking around, `SomeProject Security` is typical. Rolfe, IMHO we should keep capital `S` in `Security`. May be `Getting started with Quarkus Security ....` but we probably checked it with Rostislav, it did not show too well, I may be the guide name proved too long etc.

Reply:
Thanks. Updated. This is referring to Quarkus Security.